package.json

```diff
@@ -115,7 +115,8 @@
     "tiny-lr": "^1.0.3",
     "tree-sync": "^1.1.4",
     "walk-sync": "^0.3.0",
-    "yam": "0.0.22"
+    "yam": "0.0.22",
+    "yarn": "^0.17.8"
```

Comment:
Regarding "iterate" ... it appears to be remarkably simple to do, Ember CLI is well factored for that change.

I'm more likely to be blocked on the testing side than I am the user-facing side.